packages/jsxstyle/jsxstyle.tsx

```diff
@@ -1,21 +1,30 @@
 import * as React from 'react';
 import { componentStyles, getStyleCache } from 'jsxstyle-utils';
+import { StyleCache } from 'jsxstyle-utils/src/getStyleCache';
 
-export const cache = getStyleCache();
+export const cache: StyleCache = getStyleCache();
 
-function factory(displayName, defaultProps, tagName) {
-  tagName = tagName || 'div';
-  if (
-    process.env.NODE_ENV === 'development' &&
-    (typeof displayName !== 'string' || !displayName)
-  ) {
-    throw new Error(
-      'makeReactStyleComponentClass expects param 1 to be a valid displayName'
-    );
-  }
+export interface JsxstyleProps {
+  className?: string;
+  component?: string | React.ComponentClass | React.SFC;
```

Comment:
Use `keyof JSX.IntrinsicElements` instead of `string`.

Reply:
🙏 thank you